cli/cli.go

```diff
@@ -168,7 +168,9 @@ func Init() {
 			installCommand(args[0])
 		}
 	} else {
-		listCommand()
+		if !installFromVersionFile() {
```

Comment:
Use [`fsutil.CheckPerms`](https://godoc.org/pkg.re/essentialkaos/ek.v1/fsutil#CheckPerms) method for checking `.ruby-version` file availability instead of method invocation.